typings/custom-typings.d.ts

```diff
@@ -12,6 +12,17 @@ declare module "omit.js";
 
 declare module 'rc-animate';
 
+declare module 'rc-util/lib/Dom/addEventListener' {
```

Comment:
There are lots of methods in `rc-util` that I am not able to cover them in a short time.